### PR TITLE
fix layout on mobile dev, according to mocks

### PR DESCRIPTION
**Pull request by evoX0**
#### Please tick a box ###
- [ ] **feature** _(A new feature_)
- [x] **fix** _(A bug fix_)
- [ ] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [ ] Yes
- [ ] No

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -124,7 +124,9 @@ export { MoreComponent           } from './atoms/MainRecipe/MoreLine';
 export { KKLogo                  } from './atoms/KKLogo';
 export { ButtonWithModal         } from './atoms/ButtonWithModal';
 export { NumberedListItem        } from './atoms/NumberedListItem';
-
+export { ActivityTime	         } from './atoms/SvgIcon/ActivityTime';
+export { Difficulty	      	     } from './atoms/SvgIcon/Difficulty';
+export { TotalTime	        	 } from './atoms/SvgIcon/TotalTime';
```
Comment (misund) [inline src/index.js]: Indentation

Comment (misund) [reply src/index.js]: Use spaces for alignment

File: src/molecules/IconBar.js [hunk 2]
```diff
@@ -10,16 +10,20 @@ const PaddingLess = Col.extend`
...
+	align-items: baseline;
+`;
+
 const IconBar = props => (
-	<Row>
+	<RowBase>
 		{
 			props.entities.map((entity, i) => (
-				<PaddingLess xs key={i}>
+				<PaddingLess className="Zalupa" xs={4} key={i}>
```
Comment (misund) [inline src/molecules/IconBar.js]: zalupa

File: src/index.js (hunk 1 above)
Comment (evoX0) [reply src/index.js]: Strange. In IDE all ok

Comment (misund) [reply src/index.js]: This is alignment, and not indentation, so you should use spaces. Tabs vary in width depending on user settings.

Comment (evoX0) [reply src/index.js]: ok. got it

Comment (evoX0) [reply src/index.js]: about icons, we plan to use switch after that release

Comment (misund) [reply src/index.js]: That's OK, incremental changes are good.

I was wondering, thanks for mentioning it.

**Pull request merged**